modules/oadp-features-plugins-known-issues.adoc

```diff
@@ -49,3 +49,22 @@ You can force the reconciliation by making any minor change to the DPA itself.
 ----
 $ oc -n openshift-adp get secret/oadp-<bsl_name>-<bsl_provider>-registry-secret -o json | jq -r '.data'
 ----
+
+
+[id="openshift-adp-controller-manager-seg-fault_{context}"]
+== `openshift-adp-controller` segmentation fault
+
+The `openshift-adp-controller-manager` pod fails with a crash loop segmentation fault, when the pod crashes and is restarted indefinitely until returning a segmentation fault, when configuring a DPA with `cloudstorage` and `restic` enabled.
```

Comment:
```suggestion
If you configure a DPA with both `cloudstorage` and `restic` enabled, the `openshift-adp-controller-manager` pod crashes and restarts indefinitely until the pod fails with a crash loop segmentation fault.
```
Does this change the meaning to much? If so, I find this a bit easier to understand.

Reply:
Now that I have gotten to the workaround, I wonder if this would be more accurate?
```suggestion
When you a DPA with `cloudstorage` and `restic` enabled, if you specify values for the `velero` API, the `openshift-adp-controller-manager` pod fails with a crash loop segmentation fault.
```